plugins/inputs/opcua_client/opcua_client_test.go

```diff
@@ -0,0 +1,67 @@
+package opcua_client
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+)
+
+type OPCTags struct {
+	Name           string
+	Namespace      string
+	IdentifierType string
+	Identifier     string
+	DataType       string
+	Want           string
+}
+
+func TestClient1(t *testing.T) {
+	var testopctags = []OPCTags{
+		{"ProductName", "0", "i", "2261", "string", "open62541 OPC UA Server"},
+		{"ProductUri", "0", "i", "2262", "string", "http://open62541.org"},
+		{"ManufacturerName", "0", "i", "2263", "string", "open62541"},
+	}
+
+	var o OpcUA
+	var err error
+
+	o.Name = "testing"
+	o.Endpoint = "opc.tcp://opcua.rocks:4840"
```

Comment:
this is hitting a remote service on the internet? can we store and mock the response instead?